tests/contrib/operators/test_kubernetes_job_operator.py

```diff
@@ -0,0 +1,116 @@
+from airflow.models import DAG, TaskInstance
+from datetime import datetime, timedelta
+from airflow.utils.kubernetes_utils import KubernetesContainerInformation
+import airflow.contrib.operators.kubernetes_operator as kubernetes_operator
+import mock
+import unittest
+
+
+class KubernetesJobOperatorTest(unittest.TestCase):
+
+    ARGS = {
+        'owner': 'jessica'
+    }
+    DAG = DAG(
+        dag_id='DAG_ID',
+        default_args=ARGS,
+        schedule_interval="17 1 * * *",
+        start_date=datetime(2019, 10, 1),
+        catchup=False
+    )
+    TASK_ID = 'test-task-id'
+    YAML = """
+        apiVersion: batch/v1
+        kind: Job
+        metadata:
+          name: {{ job_name }}
+        spec:
+          template:
+            spec:
+              containers:
+              - name: test-container-name
+                image: test-container-image-information
+                command: ['test', 'command']
+              restartPolicy: Never
+          backoffLimit: 0
+    """
+    CONTAINER = KubernetesContainerInformation(name='test-container-name',
+                                               image='test-container-image-information',
+                                               command=['test', 'command'])
+
+    SUCCESS_SIDE_EFFECTS = ['applied',
+                            '1 Running / 0 Succeeded / 0 Failed',
+                            '0 Running / 1 Succeeded / 0 Failed',
+                            'cleaned up']
+
+    FAILURE_SIDE_EFFECTS = ['applied',
+                            '1 Running / 0 Succeeded / 0 Failed',
+                            '0 Running / 0 Succeeded / 1 Failed',
+                            'cleaned up']
+
+    def setUp(self):
+        kubernetes_operator.SLEEP_TIME_BETWEEN_POLLING = 1
```

Comment:
If you are going to use mock, you probably should use it to patch. You can even add `@mock.patch(..., new=1)` to the class level, so it should apply to all tests. The problem is if this test runs before some other test, that other test might require that this global variable be set to its original value.

Its actually pretty unlikely to cause problems in this case, but its a good habit to get into, since this is a common cause of tests being flaky, or previously passing then starting to fail when the order of tests changes.

Also: Can you just pass `0`?

Reply:
I never thought of that, thanks!

I can make it 0.